### ISSUE TITLE
rtf encoding danish chars

### ISSUE THREAD

**Issue by askemottelson** [deanmalmgren/textract#151]
`textract.process("test.rtf").decode(encoding='UTF-8')`
-> test זרו
[test.rtf.zip](https://github.com/deanmalmgren/textract/files/907377/test.rtf.zip)

```
res = textract.process("test.rtf").decode(encoding='UTF-8')
assert "æøå" in res
```
-> err
unrtf --text test.rtf
-->
### invalid font number 0

-----------------
test æøå

**Comment by deanmalmgren** [comment]
When I run `unrtf` on that file, my output looks like:

```
vagrant@dev:/vagrant$ unrtf --text test.rtf
###  Translation from RTF performed by UnRTF, version 0.21.5 
### font table contains 70 fonts total

AUTHOR: Aske Mottelson
### creation date: 7 February 2017 
### revision date: 7 February 2017 
### total pages: 1
### total words: 1
### total chars: 8

-----------------
test ??R
```

It doesn't look like unrtf is handling the encoding properly in this case.

**Comment by askemottelson** [comment]
Well that's true, same here -- maybe because it's zipped? If I unrtf directly on the file:

http://papirkriger.dk/test.rtf

unrtf --text test.rtf > out.txt
subl out.text
```
###  Translation from RTF performed by UnRTF, version 0.21.9 
### font table contains 0 fonts total
### invalid font number 0

-----------------
test æøå
```

**Comment by deanmalmgren** [comment]
Yeah. I unzipped it first but I still only see the question marks instead of the unicode characters, which makes me think that something is different about our environments. Do you have a shell environment variable set that might tell `unrtf` that the encoding in your RTF documents is different?

Also, its interesting but the HTML output of unrtf appears to be correct for me:

```
vagrant@dev:/vagrant$ unrtf test.rtf
<!DOCTYPE html PUBLIC -//W3C//DTD HTML 4.01 Transitional//EN>
<html>
<head>
<meta http-equiv="content-type" content="text/html; charset=utf-8">
<!-- Translation from RTF performed by UnRTF, version 0.21.5 -->
<!--font table contains 70 fonts total-->
<meta name="author" content="Aske Mottelson">
<!--creation date: 7 February 2017 -->
<!--revision date: 7 February 2017 -->
<!--total pages: 1-->
<!--total words: 1-->
<!--total chars: 8-->
</head>
<body><font size="3"><font face="Cambria">test &aelig;&oslash;&aring;</font><br>
</font></body>
</html>
```

**Comment by askemottelson** [comment]
No I know. But for some reason, when I unrtf the file after unzipping it, I get the same (wrong) output as you. 

And yes, the unrtf html output is correct. Super weird. 

It could be that unrtf is the issue, not textract, but I still don't know how we get correct encoding out of our RTF's

**Comment by askemottelson** [comment]
I've tried both on my host machine (OSX), and inside my docker, same issue

**Comment by deanmalmgren** [comment]
It looks like we could chain together unrtf + the html parser to properly handle the encoding:

```
vagrant@dev:/vagrant$ unrtf --html test.rtf > test.html
vagrant@dev:/vagrant$ textract test.html

//W3C//DTD HTML 4.01 Transitional//EN>

test æøå
```

We would obviously want to handle this better than running textract within textract, but this seems like a very plausible fix. Want to propose a PR?

Seems like [`rtf_parser.Parser`](https://github.com/deanmalmgren/textract/blob/master/textract/parsers/rtf_parser.py) could inherit from `ShellParser` (as it currently does) and also from [`html_parser.Parser`](https://github.com/deanmalmgren/textract/blob/master/textract/parsers/html_parser.py) and then adapt the `rtf_parser.Parser.extract` method accordingly

**Comment by askemottelson** [comment]
That seems like a plausible fix indeed. Let me look at it tonight. It might very well just me the --text option in unrtf that's buggy

**Comment by deanmalmgren** [comment]
Looks like there is a newer version of `unrtf` ([webpage](https://www.gnu.org/software/unrtf/)). I actually have the latest version installed in OSX but the `--text` option does not work there either. Might be good to report the bug...

**Comment by askemottelson** [comment]
Yeah, as documented [here](https://github.com/deanmalmgren/textract/issues/151#issuecomment-292720159), I actually do get the correct text output, using unrtf v. 0.21.9
